addon/components/cf-field/input/file.js

```diff
@@ -72,6 +67,7 @@ export default Component.extend({
       }
     },
 
+    @waitFor
```

Comment:
Using a test helper in a component - why?

Reply:
The `await triggerEvent(...)` even though it apparently uses `settled()` internally it doesn't actually wait for the asynchronous function to terminate. From what I've read it doesn't realize that it has to wait for that particular promise and that's where the emberjs `waitFor` test helper decorator gives it that information. Is there a solution that can be used within the test that ensures that it waits for the `save` function to terminate?

Reply:
Using the `waitUntil` test helper in the test seems to work as well.

Reply:
Seems more reasonable